closed/get_j9_source.sh

```diff
@@ -50,7 +50,7 @@ fi
 declare -A j9repos
 declare -A branches
 declare -A default_j9repos=( [openj9]=eclipse/openj9 [omr]=eclipse/openj9-omr )
-declare -A default_branches=( [openj9]=openj9-0.23.0-m2 [omr]=openj9-0.23.0-m2 )
+declare -A default_branches=( [openj9]=openj9-0.23.0 [omr]=openj9-0.23.0 )
```

Comment:
As I write this, I see neither a branch nor a tag named 'openj9-0.23.0' in openj9 and openj9-omr.

Reply:
They are coming. Do they need to exist before this gets merged?

Reply:
I suppose nobody will expect this branch to build successfully until those tags are created, so this need not wait.